data/cohort.json

```diff
@@ -4,339 +4,309 @@
-      "bio": "",
+      "reelThemIn": "Code...It's what's for dinner.",
+      "bio": "The only thing that stands between a man and what he really wants from life is often merely the will to try it and the faith to believe that it is possible. After my son was born I realized that it would be hypocritical of me to teach him to chase his dreams if I hadn't done so myself. Growing up I was always had a strong affinity for computers but I never really considered making a career out of it. After experiencing several different careers in my young adult life, I realized I would never truly be happy until I was doing something I loved. So here I am, chasing my dream one line of code at a time.",
       "github": "https://github.com/CosignMyCodesign" ,
       "linkedIn":"https://www.linkedin.com/in/dillon-williams-software-developer/",
-      "portfolio":"" ,
+      "portfolio":"https://www.dillon-williams.com" ,
       "email": "dillonwilliamsail@gmail.com",
-      "proImg": "",
-      "funImg": ""
+      "proImg": "images/classmates/DillonWilliams.jpg",
+      "funImg": "images/classmates/DillonWilliams_fun.jpg",
+      "hired": false
     },
     {
-      "id": 8,
+      "id": 9,
       "firstName": "Elyse",
       "lastName": "Dawson",
-      "reelThemIn": "",
-      "bio": "",
+      "reelThemIn": "You down with OOP? Yeah you know me.",
+      "bio": "I'm no stranger to playing many roles, both onstage and off. After years as an actor and office administrator, I turned to my love of tech for a new part. I enjoy building software that improves the quality of life for a diverse range of users and am especially interested in XR development, both practical and game related. With my background as an actor, storyteller and casual graphic designer, I believe that UX, UI, and accessibility are pivotal to an application's success.",
       "github": "https://github.com/CurtainUp" ,
       "linkedIn":"https://www.linkedin.com/in/elysedawson/",
-      "portfolio":"" ,
+      "portfolio":"https://curtainup.github.io/" ,
       "email": "dawson.elyse@gmail.com",
-      "proImg": "",
-      "funImg": ""
+      "proImg": "images/classmates/ElyseDawson.jpg",
+      "funImg": "images/classmates/ElyseDawson_fun.jpg",
+      "hired": false
     },
     {
-      "id": 9,
+      "id": 10,
       "firstName": "Jase",
       "lastName": "Hackman",
-      "reelThemIn": "",
-      "bio": "",
+      "reelThemIn": "Jase “The Hack Man” Hackman",
+      "bio": "I have always been a strange mix of creative and analytical, but have struggled to find where that duality can coexist. As a musician I am been able to explore my creativity, and in my business ventures I have been able to poor over details and strategy, but I have been unable to marry the two. As a software developer, I am able to use my whole self to solve problems. I have the opportunity to come up with a creative solution, while at the same time, dive deep into the details in order to bring that solution to life.",
```

Comment:
Well... I found a spelling error in my own bio.... and turns out it is also in my resume... darn. 

Please change poor to pore

Reply:
fixed!